web/collectBenchmarks.sh

```diff
@@ -79,7 +79,7 @@ done
 ######################################
 
 # Make SimTrack Validation directories
-simdir="SIMVAL"
+simdir="SIMVAL_MTV"
```

Comment:
@tresreid , since the contents of SIMVAL_MTV and SIMVAL_MTV_SEED will be the same, we can reduce the copy-paste, and just make a for loop over the two directories.